src/DB/projects.json

```diff
@@ -2005,5 +2005,23 @@
           "tech": ["Html", "CSS","Flask-WTF","Flask-Bootstrap"]
         }
       ]
+  },
+  {
+    "github_username": "Deepanshu0703",
+    "Social_media": {
+      "gitHub": "https://github.com/Deepanshu0703",
+      "LinkedIn": "https://www.linkedin.com/in/deepanshu-kumar-403200254",
+      "Instagram": "",
+      "YouTube": "",
+      "Twitter": ""
+    },
+    "Projects": [
+      {
+        "link": "https://github.com/Deepanshu0703/google-keeper-clone",
+        "title": "Google Keeper Clone",
+        "description": "This is a clone of google keeper app. It is a note taking app where you can add, delete and edit notes.",
+        "tech": ["ReactJs", "CSS"]
+      }
```

Comment:
hey just add the code to the previous code. you have already added your projects here. you don't need to add whole code multiple times

Reply:
Oh Sorry for that. Now corrected it